CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.25
```

Comment:
Should this be included in the backport?..

Reply:
Nope. Good catch!

Let me fix this! I need to rebase against other changes first anyway.